PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -662,9 +664,18 @@ struct HfCandidateCreator2ProngExpressions {
     for (const DeviceSpec& device : workflows.devices) {
       if (device.name.compare("hf-candidate-creator-2prong") == 0) {
         for (const auto& option : device.options) {
-          if (option.name.compare("hfEvSel.zPvPosMax") == 0) {
-            zPvPosMax = option.defaultValue.get<float>();
-            break;
+          if (option.name.compare("hfEvSel.useSel8Trigger") == 0) {
+            hfEvSelMc.useSel8Trigger = option.defaultValue.get<bool>();
+          } else if (option.name.compare("hfEvSel.useTvxTrigger") == 0) {
+            hfEvSelMc.useTvxTrigger = option.defaultValue.get<bool>();
+          } else if (option.name.compare("hfEvSel.useTimeFrameBorderCut") == 0) {
+            hfEvSelMc.useTimeFrameBorderCut = option.defaultValue.get<bool>();
+          } else if (option.name.compare("hfEvSel.useItsRofBorderCut") == 0) {
+            hfEvSelMc.useItsRofBorderCut = option.defaultValue.get<bool>();
+          } else if (option.name.compare("hfEvSel.zPvPosMin") == 0) {
+            hfEvSelMc.zPvPosMin = option.defaultValue.get<float>();
+          } else if (option.name.compare("hfEvSel.zPvPosMax") == 0) {
+            hfEvSelMc.zPvPosMax = option.defaultValue.get<float>();
```

Comment:
Seeing the increasing list, this is becoming difficult to maintain, so I would suggest adding a method for configuration:
```c++
  void configureFromDevice(const o2::framework::DeviceSpec &device)
  {
    for (const auto& option : device.options) {
      if (option.name.compare("hfEvSel.zPvPosMax") == 0) {
        zPvPosMax = option.defaultValue.get<float>();
      } else if {
        ...
      }
    }
  }
```